app/javascript/flavours/glitch/features/compose/components/options.jsx

```diff
@@ -113,7 +112,8 @@ class ToggleOption extends ImmutablePureComponent {
 
 }
 
-export default @injectIntl
+const ToggleOption = connect((state, { name }) => ({ checked: state.getIn(['compose', 'advanced_options', name]) }))(ToggleOptionIntl);
```

Comment:
Not wrong and I'm not sure if there is a reason for it, but while this is changed anyway I would prefer `(state, { name }) => (...)` to be a mapStateToProps for easier readability.

Reply:
I think the upstream 'style guide' (as it lives in Eugen's head, perhaps) does single mappings in-line, but I agree that's kind of silly and makes things harder to grok